docs/services/navigation/_index.md

```diff
@@ -15,19 +15,20 @@ The Navigation service is the stateful definition of Viam's [motion service](/se
 It uses GPS to autonomously navigate a rover [base](/components/base/) to user defined endpoints called waypoints.
 Configure your base with a navigation service, add waypoints, and set the mode of the service to [**Waypoint**](#setmode) to move your rover along a defined path at your desired motion configuration.
 
-## Configuration
+## Requirements
 
-You must configure a [base](/components/base/) with a [movement sensor](/components/movement-sensor/) as part of your robot to configure a Navigation service.
+You must configure a [base](/components/base/) with [movement sensors](/components/movement-sensor/) as part of your robot to configure a Navigation service.
 
-{{% alert title="Important" color="note" %}}
+Configure a stack of movement sensors that satisfy the following requirements in their {{< glossary_tooltip term_id="model" text="models'" >}} implementations of the [Movement Sensor API](/components/movement-sensor/#api):
```

Comment:
In a requirements section we're not telling the reader what to do, but merely requiring it. 
```suggestion
The configured movement sensors must satisfy the following requirements in their {{< glossary_tooltip term_id="model" text="models'" >}} implementations of the [Movement Sensor API](/components/movement-sensor/#api):
```

Alternatively you could do something like "To satisfy the requirements for the navigation service, you need to configure movement sensors that..."